payments-ui-core/res/values-fil/strings.xml

```diff
@@ -80,6 +80,14 @@
   <string name="stripe_pay_button_amount">Magbayad ng %s</string>
   <!-- Card details entry form header title -->
   <string name="stripe_paymentsheet_add_payment_method_card_information">Impormasyon ng kard</string>
+  <!-- Payment Method type brand name  -->
+  <string name="stripe_paymentsheet_payment_method_afterpay_clearpay">Afterpay</string>
```

Comment:
should we rename the key so it doesn't say `paymentsheet`?

Reply:
yeah, I agree, I will follow up with that in a later PR